x-pack/plugins/security_solution/server/lib/telemetry/tasks/prebuilt_rule_alerts.ts

```diff
@@ -6,14 +6,19 @@
  */
 
 import type { Logger } from '@kbn/core/server';
-import type { SortResults } from '@elastic/elasticsearch/lib/api/types';
 import type { ITelemetryEventsSender } from '../sender';
 import type { ITelemetryReceiver } from '../receiver';
 import type { ITaskMetricsService } from '../task_metrics.types';
-import type { ESClusterInfo, ESLicense, TelemetryEvent } from '../types';
+import type { TelemetryEvent } from '../types';
 import type { TaskExecutionPeriod } from '../task';
 import { TELEMETRY_CHANNEL_DETECTION_ALERTS } from '../constants';
-import { batchTelemetryRecords, processK8sUsernames, newTelemetryLogger } from '../helpers';
+import {
+  batchTelemetryRecords,
+  processK8sUsernames,
```

Comment:
Is this used?

Reply:
do you mean `processK8sUsernames`?
I think so, it was added by https://github.com/elastic/kibana/pull/152129